examples/dataset/age_estimation/age_estimation/upload_results.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from argparse import ArgumentParser
+from argparse import Namespace
+from typing import Optional
+
+import pandas as pd
+from age_estimation.constants import BUCKET
+from age_estimation.constants import DATASET
+from tqdm import tqdm
+
+import kolena
+from kolena.dataset import fetch_dataset
+from kolena.dataset import test
```

Comment:
Need to update to upload_results since https://github.com/kolenaIO/kolena/pull/390 was merged